.github/workflows/run_python_tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
```

Comment:
Could you try to add python 3.9 and 3.10? At least 3.9 should work. (3.10 should also work, but I haven't tested this locally.)